test-e2e/panels/ranges.test.ts

```diff
@@ -116,11 +116,11 @@ test('Automatic ranges detection should work', async ({ page }) => {
   );
 
   const rangesData = [
-    { s: '1.75 - 1.79', r: '0.49' },
-    { s: '1.95', r: '2.05' },
-    { s: '1.94', r: '2.05' },
-    { s: '2.15', r: '0.07' },
-    { s: '2.31 - 2.34', r: '1.01' },
+    { s: '1.75 - 1.79', r: '3.49' },
+    { s: '1.95', r: '14.67' },
+    { s: '1.94', r: '14.67' },
+    { s: '2.15', r: '0.53' },
+    { s: '2.31 - 2.34', r: '7.24' },

```

Comment:
We should not fixed the test bug fix the bug that has introduced this changes.

Before we were automatically setting has sum for the ranges the number of H that were present in the molfile. This is still working correctly in the last release that is currently working on www.nmrium.org.

But in this release the sum is not automatically calculated. @hamed-musallam this may have been introduced by the changes of @jobo322. Please check were it should be corrected. In nmr-load-save ?

![2022-10-16 07 37 53](https://user-images.githubusercontent.com/1484241/196020039-5037dfbb-4d73-478d-8b21-200f26e2cce3.gif)

Reply:
the default value from nmr-load-save is 100

https://github.com/cheminfo/nmr-load-save/blob/08dcd987ab042d5d1de62a5512dd5cffbb930755/src/utilities/formatSpectrum1D.ts#L74-L79

Reply:
all these options should be handed by NMRium and not by the nmr-load-save package. what do you think?

Reply:
to make it work the sumAuto: true option should be in the ranges options.

in NMRium Once we initiate the 1D spectrum we initiate the ranges object with the default options if it is not provided in the JSON file.

```ts
{
sumAuto:true,
sum: undefined
}
```

Reply:
@lpatiny 

what do you think we have to do in this case, should be all the options on the side nmr-load-save, or should be NMRium which I think it is better because it directly related to NMRium view

Reply:
we need to add comments in the test to clarify this behavior